lib/kubeclient.rb

```diff
@@ -22,8 +22,12 @@ class Client
       [Kubeclient.const_set(et, Class.new(RecursiveOpenStruct)), et]
     end
 
-    def initialize(uri, version)
+    def initialize(uri, version = 'v1beta3')
       @api_endpoint = (uri.is_a? URI) ? uri : URI.parse(uri)
+
+      unless @api_endpoint.path.include? 'api'
+        @api_endpoint.merge!(File.join(@api_endpoint.path, 'api'))
```

Comment:
> You probably don't want this because File.join will use the default file separator which is \ on Windows. I would just use string interpolation.

good point, thanks !
I assume the same goes for line 31 too (which is not part of this patch though)

BTW from looking at the documentation of the method it's really unclear...